pywren/wrenconfig.py

```diff
@@ -60,6 +63,17 @@ def get_default_config_filename():
 
     return config_filename
 
+def patch_storage_config(config_data):
```

Comment:
Can we print a warning or something here ? I'm trying to understand if this is an unusual case where we need to tell the user that we are using s3 as the default storage or if this is an expected workflow